application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/config/connection/AgentConnectionConfig.java

```diff
@@ -37,6 +37,7 @@ public final class AgentConnectionConfig
     private RackAware myRackAware = new RackAware();
     private HostAware myHostAware = new HostAware();
     private Class<? extends DefaultLoadBalancingPolicy> myDatacenterAwarePolicy = DataCenterAwarePolicy.class;
+    private String myInstanceName;
```

Comment:
Should it have a default value? Like using the hostname/hostid

Reply:
I have made it mandatory and it would throw configuration error if someone will not set it. if we use default value it will defeat that purpose. so it is mandatory property and must be set.